.github/workflows/test.yml

```diff
@@ -143,6 +143,7 @@ env:
   CONTACT_US_EMAIL_ADDRESS: ${{ vars.CONTACT_US_EMAIL_ADDRESS }}
   FF_TFM_PAYMENT_RECONCILIATION_ENABLED: "true"
   FF_TFM_FACILITY_END_DATE_ENABLED: "true"
+  FF_TFM_DEAL_CANCELLATION: "false"
```

Comment:
I've set it to false here as the default values on the github workflows as I think that's what Abhi wants to move to. It shouldn't impact the unit tests as these mock the value anyway

Reply:
We've got a new pattern for working with feature flags -- see https://github.com/UK-Export-Finance/dtfs2/pull/3390/commits. Might be worth having a chat with Abhi about how to get this up to the new pattern (I think that should be done in this PR -- and I do think that we should look at maybe creating the e2e-test here for the button to build up over the next few tickets so we can establish the new FF e2e pattern)

Of course -- I'm not dead set on this and let me know if you disagree or want to chat about ti

Reply:
Youll need to add this to the docker files too :)

Reply:
Oh yep, didn't realise Abhi already had a PR open for it. I'll look into it!

Reply:
Have refactored to the new pattern. I'm going to leave e2e tests out of this PR to help get this PR in quickly as it's a bit of a blocker on other work. I'll add as a subtask to the ticket to add them in separately.